code/lib/blocks/src/controls/Boolean.tsx

```diff
@@ -88,6 +93,11 @@ const Label = styled.label(({ theme }) => ({
     color: theme.color.defaultText,
     padding: '7px 15px',
   },
+  '@media (forced-colors: active)': {
```

Comment:
If I remember correctly, you can nest media queries in the above selector (below line 94) which should result in not needing to duplicate a complex selector like this one. It's just slightly tidier.

Reply:
Media queries are now nested.
Applied to other changes as well.